src/policies/exemptions/security.ts

```diff
@@ -0,0 +1,12 @@
+import { registerExemptions } from ".";
+import { neuvector, promtail } from "./matchers";
+
+export const exemptDropAllCapabilities = registerExemptions([
+  neuvector.enforcer,
+  neuvector.prometheus,
+]);
```

Comment:
Curious on these exemptions - should be able to drop all for these ones too, I don't think there is anything by default in the upstream or BB chart that would make this not-possible. We may have to `add` some capabilities but drop itself should be fine.